js/client/modules/@arangodb/test-utils.js

```diff
@@ -341,9 +341,13 @@ function performTests (options, testList, testname, runFn, serverOptions, startS
         } else {
           serverDead = true;
           continueTesting = false;
+          let msg = '';
+          if (results[te].hasOwnProperty('message') && results[te].message.len > 0) {
+            msg = results[te].message + ' - ';
+          }
           results[te] = {
             status: false,
-            message: 'server is dead: + ' + results[te].message
+            message: 'server is dead: ' + msg + instanceInfo.message
```

Comment:
How can `results[te].message` not have `results[te].message.length > 0`? Where is it set?

Reply:
If there has been a test-result with a failure, this may still be in there.